app/assets/javascripts/discourse/app/components/about-page-user.gjs

```diff
@@ -0,0 +1,70 @@
+import Component from "@glimmer/component";
+import { service } from "@ember/service";
+import { htmlSafe } from "@ember/template";
+import { renderAvatar } from "discourse/helpers/user-avatar";
+import { prioritizeNameInUx } from "discourse/lib/settings";
+import { userPath } from "discourse/lib/url";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class AboutPageUser extends Component {
+  @service siteSettings;
+
+  get template() {
+    const user = this.args.user;
+    return {
+      name: user.name,
+      username: user.username,
+      userPath: userPath(user.username),
+      avatar: renderAvatar(user, {
+        imageSize: "large",
+        siteSettings: this.siteSettings,
+      }),
+      title: user.title || "",
+      prioritizeName: prioritizeNameInUx(user.name),
+    };
+  }
```

Comment:
Using an intermediary getter like this means that if even one of the referenced props changes - it will rerender everything.

Reply:
I'm not sure I understand. Do you mean that if somehow, e.g., `user.name` changes at runtime, the whole template will re-render? Even if `user.name` isn't `@tracked`?

Reply:
it doesn't need to be `@tracked` - it could be updated with `set()`. But it could also be a site setting update pushed via message bus.

But, even worse 😅, because this getter isn't cached it's being evaluated every time `this.template` is referenced in the template.

Unless there's a specific reason to do it differently - stuff should be referenced directly, e.g. `{{@user.name}}`, `{{userPath @user.username}}`